plane_default.yaml

```diff
@@ -0,0 +1,21 @@
+DOWNLOADS:
+  URL: "https://b2share.fz-juelich.de/api/files/a423a576-220d-47b0-9e0c-b5b32d45fc59"
+  URL_BACKUP: "https://zenodo.org/records/10390573/files"
+
+FASTSURFERCNN:
+  CKPT:
+    AXIAL: "FastSurferCNN/checkpoints/aparc_vinn_axial_v2.0.0.pkl"
+    CORONAL: "FastSurferCNN/checkpoints/aparc_vinn_coronal_v2.0.0.pkl"
+    SAGITTAL: "FastSurferCNN/checkpoints/aparc_vinn_sagittal_v2.0.0.pkl"
+  CFG:
+    AXIAL: "FastSurferCNN/config/FastSurferVINN_axial.yaml"
+    CORONAL: "FastSurferCNN/config/FastSurferVINN_coronal.yaml"
+    SAGITTAL: "FastSurferCNN/config/FastSurferVINN_sagittal.yaml"
+
+CEREBNET:
```

Comment:
Maybe the CerebNet paths can also  be in a different file, if you want. But the URL for CerebNet is also different.